test/accountSpec.js

```diff
@@ -11,10 +11,29 @@ const dbHelper = require('./helpers/db')
 const appHelper = require('./helpers/app')
 const logHelper = require('five-bells-shared/testHelpers/log')
 
+const accountValidator = require('../src/services/validator').create('Account')
+const transferValidator = require('../src/services/validator').create('Transfer')
 const publicKey = fs.readFileSync(__dirname + '/data/public.pem', 'utf8')
 
 const START_DATE = 1434412800000 // June 16, 2015 00:00:00 GMT
 
+function validAccounts (res) {
+  const accounts = Array.isArray(res.body) ? res.body : [res.body]
```

Comment:
It's actually only an array on `.get('/accounts')`. In other cases, it's a single account.